packages/connect/src/ExtensionProvider/ExtensionProvider.ts

```diff
@@ -89,6 +89,15 @@ export class ExtensionProvider implements ProviderInterface {
    */
   healthPingerInterval = CONNECTION_STATE_PINGER_INTERVAL
 
+  #provider = {
+    send: (message: ToExtension): void => {
+      window.postMessage(message, "*")
+    },
+    listen: (handler: ProviderListenHandler): void => {
+      window.addEventListener("message", handler)
+    },
+  }
```

Comment:
Do we need this? Why not just inline the call to `postMessage` and `addEventListener`?

Reply:
I started with that but I decided to reuse functions instead of repeating `window.postMessage` etc etc... Makes lines clearer.

I don't think this should block the PR.

Reply:
I started with that but I altered it to using a function as it makes reduces repeating `window` etc again and again.

I dont think this should block the PR

Reply:
I agree with @tomaka, I would much rather having it inlined, for different reasons:

1) It makes the next change (the removal of ExtensionMessage and ProviderMessage interfaces) a lot easier.
2) IMO it's a bogus abstraction that adds a layer of indirection for no good reason.
3) It's not true that it makes the code more concise, as the current git diff is `+51 -56` while the git diff of its inlined cunter part would be `+34 -63`.

Since @wirednkod would be traveling and he has asked me to take over, I will force push those 2 commits with the inlined code.